README.md

```diff
@@ -31,6 +31,26 @@ Table below may help you to choose a proper module for your requirements:
 | expirationd   | Medium (sec)  | Yes          | Yes                 | synchronous (fiber with condition) |
 | moonwalker    | NA            | No           | Yes                 | asynchronous (using crontab etc)   |
 
+### Prerequisites
+
+* Tarantool 1.10+ (`tarantool` package, see [documentation](https://www.tarantool.io/en/download/)).
```

Comment:
It's a bit inconsistent with phrase above:
```
it's an option box.cfg.replication_source (box.cfg.replication for Tarantool 1.7.6+)
```

So... You need to drop 1.10- mentions from readme to use 1.10+ as prerequisite

Reply:
Hmm... I don't want to declare work with 1.7.6 because we don't test it on CI.
However, 1.7.6 should work too.

Reply:
If you can't guarantee 1.7.6, let's remove it from readme.

Reply:
removed